src/index.html

```diff
@@ -9,9 +9,174 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="group_Of_Information">
+        <legend>Personal information</legend>
+
+        <label class="form_field">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          How old are you?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label class="form_field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="term"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group_Of_Information">
+        <legend>Registration</legend>
+
+        <label class="form_field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form_field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="8"
+            maxlength="32"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group_Of_Information">
+        <legend>An interesting fact about You!</legend>
+        <div class="form_field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="love_cats"
+              value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="love_cats"
+              value="no"
+            >
```

Comment:
![Screenshot 2023-01-23 at 18 50 11](https://user-images.githubusercontent.com/84275496/214099673-a0f60b8e-393c-4691-a94a-95b7369df992.png)

Reply:
I resolved all mistakes and created a new commit by updating the HTML file.